MekHQ/src/mekhq/campaign/universe/enums/HiringHallLevel.java

```diff
@@ -18,8 +18,12 @@
  */
 package mekhq.campaign.universe.enums;
 
+import com.fasterxml.jackson.core.JsonParser;
+import com.fasterxml.jackson.databind.DeserializationContext;
+import com.fasterxml.jackson.databind.deser.std.StdDeserializer;
 import io.sentry.Sentry;
```

Comment:
No need to include. There should be no place within the code that needs this outside of Main and the Logger.

Reply:
Yeah, whoops that was a relic from when I using a custom deserializer because I didn't realize Jackson would get it right if you fed in the raw enum value. For some reason Intellij is not showing me unused imports, which I thought it used to do.

Reply:
I think it's a setting you have to change. This stuff is always flagged on my VSCode but I can't rely on it to do MM work on it. Project is big enough there's a 50/50 chance it'll work right.